doc/man3/BIO_should_retry.pod

```diff
@@ -36,7 +36,7 @@ BIO_should_read() is true if the cause of the condition is that a BIO
 needs to read data.
 
 BIO_should_write() is true if the cause of the condition is that a BIO
-needs to read data.
+needs to write data.
```

Comment:
Trouble here (and in previous bullet) is that it's unclear what reader is supposed to do upfront. Reader as "human that reads the sentence". Yes, it's explained later on, but question also is what if reader needs to quickly refresh the memory. Thing is that with this formulation user is likely to be forced to [frustratingly enough] read whole thing over again. Because "needs to write data" can be as well be interpreted as application has to pass more data. But then it stops making sense, because these are the calls that application makes when it should *retry* last operation. And it gets counter-intuitive when it's outcome of read operation. Are you supposed to issue some write operation? [And of course vice versa.] Maybe it would be more appropriate to put it as "BIO has pending data to write, check for writability and/or retry the current operation"? Expectation is that "check for writability" would trigger enough associations with most common case of waiting for socket to become writable. Arguably even BIO_should_read might need similar adjustment. Though one can't put it as "pending data to read", it would have to be "BIO has insufficient amount of input data or non-application data to process, check for readability and/or retry current operation."